api/.eslintrc.json

```diff
@@ -47,7 +47,7 @@
       }
     },
     {
-      "files": ["src/client/*", "src/db/*", "src/client/dynamics/*"],
+      "files": ["src/client/*", "src/db/*", "src/client/dynamics/**"],
```

Comment:
The dynamics folder used to be flat but now is expected to contain potentially many subfolders, which is why this was needed.